docs/public/operators-on-minikube.adoc

```diff
@@ -0,0 +1,204 @@
+$ kubectl get og -n operators
+NAME               AGE
+global-operators   3m37s
+----
+
+If you don't see one, create it using below command:
+[source,sh]
+----
+$ kubectl create -f - << EOF
+apiVersion: operators.coreos.com/v1
+kind: OperatorGroup
+metadata:
+  name: global-operators 
+  namespace: operators 
+spec:
+  targetNamespaces:
+  - operators
+EOF
+----
+
+Now, install the etcd Operator using below command:
+[source,sh]
+----
+$ kubectl create -f - << EOF
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: etcd
+  namespace: operators
+spec:
+  channel: clusterwide-alpha
+  name: etcd
+  source: operatorhubio-catalog
+  sourceNamespace: olm
+  startingCSV: etcdoperator.v0.9.4-clusterwide
+  installPlanApproval: Automatic
+EOF
+----
+
+Give it a few seconds before checking the availability of the etcd Operator:
+[source,sh]
+----
+$ odo catalog list services
+Operators available in the cluster
+NAME                                CRDs
+etcdoperator.v0.9.4-clusterwide     EtcdCluster, EtcdBackup, EtcdRestore
+----
+
+==== Troubleshooting
+
+If you don't see etcd Operator using above command or by doing `kubectl get csv -n operators`, make sure that pod belonging to the `CatalogSource` named `operatorhubio-catalog` is running:
+[source,sh]
+----
+$ kubectl get po -n olm | grep operatorhubio-catalog
+----
+
+If the state of this pod is `CrashLoopBackOff`, delete it so that Kubernetes will automatically spin up a new pod for the `CatalogSource`:
+
+[source,sh]
+----
+$ kubectl delete po -n olm <name-of-operatorhubio-catalog-pod>
+----
+
+Once the pod for this `CatalogSource` is up, wait a few seconds before trying to find the etcd Operator when you do `odo catalog list services`.
+
+=== Installing the Service Binding Operator
+
+We use the link:https://github.com/redhat-developer/service-binding-operator/[Service Binding Operator] to provide `odo link` feature which links an odo component with an Operator backed service. Thus, to be able to use this feature, it's essential that we install the Operator first.
+
+Service Binding Operator is not yet available via the OLM. The team is link:https://github.com/redhat-developer/service-binding-operator/issues/727[working on making it available] through OLM.
+
+At the moment, to make Service Binding Operator work on Kubernetes, we need to keep a couple of things in mind:
+
+. It needs to be installed by cloning its GitHub repository
+. It must be installed in an individual namespace unlike the cluster wide installation we saw for etcd Operator in previous section
+
+
+Let's setup the Service Binding Operator now.
+
+. First, let's clone its repository on the host system where we're running minikube and checkout the specific tag that is currently supported by `odo link` command:
++
+[source,sh]
+----
+$ git clone https://github.com/redhat-developer/service-binding-operator/
+$ cd service-binding-operator
+$ git fetch --tags
+$ git checkout tags/v0.1.1-364 -b v0.1.1-364
```

Comment:
Could we replace these with a single long yaml file ? Otherwise, for an `odo` user, this is a lot of work :( 
Ideally, every release in Service Binding should have a "long" yaml as a release artifact. 

@pedjak @Avni-Sharma

Reply:
@sbose78 @pedjak Would it make sense for SBO to provide one big yaml file (or multiple files if you prefer) with all resources that are required for SBO installation? The users could then just do one simple `kubect create -f ` and get the whole setup?
A good example is how OLM does it https://github.com/operator-framework/operator-lifecycle-manager/releases/tag/0.16.1

Reply:
+1 @kadel , I would prefer that too.

Reply:
I am on +1 on that as well, but before publishing such yaml file, we should run our acceptance tests to verify if the operator is going to work really. We are working on making our acceptance tests runnable agains plain k8s clusters. cc @pmacik